shop/README.md

```diff
@@ -0,0 +1,27 @@
+# Shop
```

Comment:
I'd clear this readme. Its generated and not really specific to your project. A lot of this stuff an angular dev would already know so only add instructions that help your team